app/(app)/family-space/page.tsx

```
@@ -87,22 +165,33 @@ export default function FamilySpacePage() {
-        });
+    if (window.Kakao && window.Kakao.isInitialized()) {
+      const baseUrl = process.env.NEXT_PUBLIC_BASE_URL || window.location.origin;
+      const imageUrl = `${baseUrl}/images/modi-logo-small.png`;
+
+      window.Kakao.Link.sendDefault({
+        objectType: 'feed',
+        content: {
+          title: `🌱 ${family.family.name} 가족 스페이스에 초대합니다!`,
+          description: `함께 식물을 키우고 요금제도 절약해요!\n초대 코드: ${family.family.inviteCode}`,
+          imageUrl: imageUrl,
+          link: {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)